docs/docs/overview/quickstart/get-started-with-java.md

```diff
@@ -0,0 +1,336 @@
+---
+sidebar_position: 12
+sidebar_label: Get started with Java
+---
+
+# Java Quickstart
+
+Learn how to integrate Novu into your Java project on the fly and send notifications across different channels (SMS, Email, Chat, Push).
+
+In this Quickstart, you will learn how to:
+
+- [Install and Set up the Novu Java SDK.](#install-and-set-up-the-novu-java-sdk-in-your-project)
+- [Configure a Notification Channel Provider.](#set-up-a-channel-provider)
+- [Set up a Notification Workflow.](#create-a-notification-workflow)
+- [Create a Subscriber and update a Subscriber's information.](#create-a-subscriber-and-update-a-subscribers-information)
+- [Send your first Notification to a Subscriber.](#trigger-a-notification)
+- [Send Notifications via Topics.](#topics)
+
+Let's get started! :muscle:
+
+## Requirements
+
+To be able to use this Quickstart, you would need to have the following:
+
+- A Novu account, if you don't have one yet, [sign up](https://web.novu.co) for free.
+- A working Java development environment with JDK 17 and Springboot Framework of 4.0.0+.
```

Comment:
The Spring version I can see in the quickstart code is v3.1.2 but you mentioned v4.0 here. I think you should reconcile this

Reply:
Fixed